transaction-history-manager/src/main/java/com/qpay/transactionhistorymanager/controller/TransactionHistoryController.java

```diff
@@ -0,0 +1,37 @@
+package com.qpay.transactionhistorymanager.controller;
+
+import com.qpay.transactionhistorymanager.model.dto.TransactionModification;
+import com.qpay.transactionhistorymanager.model.entity.TransactionEntity;
+import com.qpay.transactionhistorymanager.service.TransactionHistoryService;
+import com.qpay.transactionhistorymanager.utility.PathUtils;
+import jakarta.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping(path = PathUtils.HISTORY_PATH)
+public class TransactionHistoryController {
+
+    private final TransactionHistoryService transactionHistoryService;
+
+    @GetMapping("{userId}")
+    public List<TransactionEntity> getPageOfTransactionsByUserId(@PathVariable final long userId,
```

Comment:
Let's make life easier
```suggestion
        public List<TransactionEntity> getPageOfTransactionsByUserId(@PathVariable final long userId, final Pageable pageable) {
```